requirements/constraints.txt

```diff
@@ -0,0 +1,9 @@
+# Version constraints for pip-installation.
+#
+# This file doesn't install any packages. It specifies version constraints
+# that will be applied if a package is needed.
+#
+# When pinning something here, please provide an explanation of why.
+
+# six is at 1.12.0, but transifex-client requires ==1.11.0
```

Comment:
Maybe include a link to the transifex-client issue you filed, for quick verification of whether it's been fixed yet or not?

Reply:
Good idea! Done.